test/sysinfo.jl

```diff
@@ -56,7 +56,18 @@ end
 end
 
 @testset "Base.Sys docstrings" begin
-    undoc = Docs.undocumented_names(Sys)
-    @test_broken isempty(undoc)
-    @test undoc == [:CPU_NAME, :JIT, :cpu_info, :cpu_summary]
+    @test isempty(Docs.undocumented_names(Sys))
+end
+
+@testset "show" begin
+    example_cpus = [Base.Sys.CPUinfo("Apple M1 Pro", 2400, 0x000000000d913b08, 0x0000000000000000, 0x0000000005f4243c, 0x00000000352a550a, 0x0000000000000000)
+    Base.Sys.CPUinfo("Apple M1 Pro", 2400, 0x000000000d9040c2, 0x0000000000000000, 0x0000000005d4768c, 0x00000000356b3d22, 0x0000000000000000)
+    Base.Sys.CPUinfo("Apple M1 Pro", 2400, 0x00000000026784da, 0x0000000000000000, 0x0000000000fda30e, 0x0000000046a731ea, 0x0000000000000000)
+    Base.Sys.CPUinfo("Apple M1 Pro", 2400, 0x00000000017726c0, 0x0000000000000000, 0x00000000009491de, 0x0000000048134f1e, 0x0000000000000000)]
+
+    @test repr(example_cpus[1]) == "Base.Sys.CPUinfo(\"Apple M1 Pro\", 2400, 0x000000000d913b08, 0x0000000000000000, 0x0000000005f4243c, 0x00000000352a550a, 0x0000000000000000)"
+    @test repr("text/plain", example_cpus[1]) == "Apple M1 Pro: \n        speed         user         nice          sys         idle          irq\n     2400 MHz    2276216 s          0 s     998861 s    8919667 s          0 s"
```

Comment:
Tests failed on windows:

```
Error in testset sysinfo:
Test Failed at C:\buildkite-agent\builds\win2k22-amdci6-2\julialang\julia-master\julia-a391a4e18f\share\julia\test\sysinfo.jl:69
  Expression: repr("text/plain", example_cpus[1]) == "Apple M1 Pro: \n        speed         user         nice          sys         idle          irq\n     2400 MHz    2276216 s          0 s     998861 s    8919667 s          0 s"
   Evaluated: "Apple M1 Pro: \n        speed         user         nice          sys         idle          irq\n     2400 MHz  227621640            0     99886140    891966730            0  ticks" == "Apple M1 Pro: \n        speed         user         nice          sys         idle          irq\n     2400 MHz    2276216 s          0 s     998861 s    8919667 s          0 s"
```

Formatted:
```
Evaluated: 
"Apple M1 Pro: \n
        speed         user         nice          sys         idle          irq\n
             2400 MHz  227621640            0     99886140    891966730            0  ticks" == 
"Apple M1 Pro: \n
        speed         user         nice          sys         idle          irq\n
             2400 MHz    2276216 s          0 s     998861 s    8919667 s          0 s"
```
The output of `Sys.cpu_summary` on Windows is inconsistent with other platforms,
and it looks like a unit issue.
No unit/ticks v.s. `s`

Maybe just skip this test on Windows (`Sys.iswindows()`) and open an issue to track this down.

Reply:
Should be fixed by https://github.com/JuliaLang/julia/pull/52777/files#r1454308561

Reply:
Also filed #52936 to use a nonzero clock tick on Windows.

Reply:
i686-w64-mingw32 passed test.
There may be something wrong with buildkite-agent.exe on "build x86_64-w64-mingw32".
But is unrelated to this pr.